examples/src/java/org/apache/heron/examples/api/AckingTopology.java

```diff
@@ -60,6 +60,8 @@ public static void main(String[] args) throws Exception {
     Config conf = new Config();
     conf.setDebug(true);
 
+    conf.setSerializationClassName(Config.HERON_KRYO_SERIALIZER_CLASS_NAME);
+
```

Comment:
It would be good to create an issue for us to try and explain in more detail how Kryo fits into and works with the Heron system.  I think it's a bit of a black box at the moment.  Doesn't need to be done with this PR.